ui/perfherder/helpers.js

```diff
@@ -550,6 +551,11 @@ export const getFrameworkData = props => {
   return { id: 1, name: 'talos' };
 };
 
+export const getFrameworkName = (frameworks, frameworkId) => {
+  const framework = frameworks.find(item => item.id === frameworkId);
+  return framework ? framework.name : legendCardText.unknownFrameworkMessage;
```

Comment:
We should make a standalone constant out of `legendCardText.unknownFrameworkMessage`, because now we're using its text in 2 distinct places, not just `LegendCard` component.

Reply:
Done. Now we have only a generic `unknownFrameworkMessage`.